scss/_containers.scss

```diff
@@ -44,4 +38,25 @@
       }
     }
   }
+
+  // OUDS mod
+  // scss-docs-start containers-max
+  .max-width-public-website {
+    @include media-breakpoint-up(2xl) {
+      --#{$prefix}container-margin-x: #{$ouds-grid-2xl-margin}; // 80px
+      max-width: $ouds-grid-2xl-max-width-public-website; // 1680px
+
+      .row {
+        --#{$prefix}gutter-x: #{$ouds-grid-2xl-column-gap}; // 32px
+      }
+    }
+  }
+
+  .max-width-specific-tools {
```

Comment:
We have the `.mw-100` utility, shouldn't we name these grid utilities `.mw-*` too?